src/diffusers/models/transformer_2d.py

```diff
@@ -213,22 +213,28 @@ def __init__(
 
     def forward(
         self,
-        hidden_states,
-        encoder_hidden_states=None,
-        timestep=None,
-        class_labels=None,
-        cross_attention_kwargs=None,
+        hidden_states: Tensor,
```

Comment:
```suggestion
        hidden_states: torch.Tensor,
```
to align with https://github.com/huggingface/diffusers/blob/b3c437e009b1efc44c56f52e62c298ff9acd727c/src/diffusers/models/prior_transformer.py#L112

Reply:
restored namespace qualifier on all uses of *Tensor symbols